src/sentry/tsdb/snuba.py

```diff
@@ -150,14 +218,24 @@ def trim(self, result, groups, keys):
                         del result[rk]
 
     def get_range(self, model, keys, start, end, rollup=None, environment_ids=None):
+        model_columns = self.all_model_columns.get(model)
+
+        if model_columns is None:
+            raise Exception(u"Unsupported TSDBModel: {}".format(model.name))
+
+        if model_columns.dataset == snuba.Dataset.Outcomes:
+            aggregate_function = "sum"
+        else:
+            aggregate_function = "count()"
```

Comment:
Any reason not to make the aggregate_function a field in the namedtuple above? 
This condition seems quite ad-hoc and easy to break.

Reply:
My reasoning here was that these aggregations work for `get_range` but not the other functions in this file. So, felt it was more appropriate to do that logic in this function.